### DISCUSSION TITLE
How to reset form fields inside the afterStateUpdated ?

### DISCUSSION THREAD

**Discussion by hazwan91** [filamentphp/filament#12863] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I have 2 radio buttons and have different kind of fields for each of them but some of it are the same. I notice that everytime i switch either one of it, some field that already been filled are not reset.

I read the docs and found the reset method `$this->form->fill()`. However i need to use it inside the `afterStateUpdated` at Radio form schema which also means $this won't be usable. So then how should i reset the forms in one line of code? I don't want use $set for all the field.

**Comment by leandrocfe** [comment, marked as answer]
`afterStateUpdated(function($livewire){ $livewire->form->fill();})`

?
